Assets/HoloToolkit/Utilities/Scripts/WorldAnchorManager.cs

```diff
@@ -136,6 +137,40 @@ public void RemoveAnchor(GameObject gameObjectToUnanchor)
         }
 
         /// <summary>
+        /// Removes all anchors from the scene and deletes them from the anchor store.
+        /// </summary>
+        public void RemoveAllAnchors()
+        {
+            SpatialMappingManager spatialMappingManager = SpatialMappingManager.Instance;
```

Comment:
It's no big deal, I was just wondering if there was a purpose is all.

I think it's fine.